### PR TITLE
add method update for id in the file UserController.java

### PR DESCRIPTION
**Pull request by DanteIturri**
Add method for the udate user for id and annotation Path 
response with error 404

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controller/UserController.java [hunk 1]
```diff
@@ -19,4 +19,15 @@
         this.userService.deleteUserById(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    @PatchMapping("/user/{id}")
+    public ResponseEntity<UserDTO> update( @PathVariable String id) {
+        Optional<UserEntity> entities = userRepository.findById(id);
+        if (entities.isPresent()) {
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/UserController.java:25]: I believe this validation should be in UserService implementation, is there any change in the UserService interface?

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/controller/UserController.java]: change done

File: src/main/java/com/alkemy/ong/controller/UserController.java [hunk 2]
```diff
@@ -19,4 +19,15 @@
         this.userService.deleteUserById(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    @PatchMapping("/user/{id}")
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/UserController.java:33]: Did you add the PatchMapping to the rout in the security config?

Comment (DanteIturri) [reply src/main/java/com/alkemy/ong/controller/UserController.java:33]: not add pathMapping, change now and oder modified

**Review by juan1977lopez — APPROVED** [review]
Approved

**Pull request merged**